plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -170,6 +170,9 @@ protected QueryRunner createQueryRunner()
     protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
```

Comment:
The PR is missing a test where Trino views for Delta gets created on a Glue backed metastore.
Consider either such a test either in `trino-delta-lake`.

Reply:
Test added